readme.md

```diff
@@ -0,0 +1,182 @@
+# Robotic Vision Integration Package (RVIP) Documentation
+![Build Status](https://img.shields.io/gitlab/pipeline/cardboardcode/rvip)
```

Comment:
Should this be updated to point to `ros-industrial/rvip` instead? And be a Travis CI badge?

Reply:
Updated.
[commit](https://github.com/cardboardcode/rvip-1/commit/24510326fbf974ee35ee5f9423b779dcb4ed6b1c)